client/my-sites/media-library/list.jsx

```diff
@@ -280,7 +259,6 @@ export class MediaLibraryList extends React.Component {
 				getItemRef={ this.getItemRef }
 				renderItem={ this.renderItem }
 				renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
-				renderTrailingItems={ this.renderTrailingItems }
```

Comment:
`SortedGrid` doesn't seem to use the `renderTrailingItems` prop. Did we remove this already?

Reply:
It's part of `<InfiniteList/>`, which `<SortedGrid/>` passes through.